README.md

```diff
@@ -6,63 +6,63 @@
 
 
 
-## What this is
+## Universal Idea
 
-Mandelbulber generates 3-dimensional fractals.
+In summary, Mandelbulber generates 3-dimensional fractals.
 
-Explore trigonometric, hypercomplex, mandelbox, IFS and many other 3D fractals.
+Explore trigonometric, hyper-complex, Mandelbox, IFS, and many other 3D fractals.
 
-Render with a great palette of customized shaders and create stunning images and videos.
```

Comment:
English noun "shaders" raises ambiguity.
We should use "shade" as a verb.
http://www.ling.upenn.edu/~rnoyer/courses/103/Ling10309Pragmatics.pdf